### ISSUE TITLE
Problem selecting alarm clock at 12:00PM-1:00PM

### ISSUE THREAD

**Issue by adnanlogicon** [fennifith/TimeDatePicker#2]
Scenario:
Selecting alarm at 12:30PM while my current time is 12:05PM  for the next day(23hour remaining). It shouldn't be (25 minutes remaining)

Solution: I fixed that issues, may i create a pull request. 
Thanks

**Comment by fennifith** [comment]
Yes! Please do. Sorry, I've been meaning to write contributing guidelines for this (and some other projects) for a while, I just haven't gotten around to it yet. Thank you for helping!
